src/Models/RETSError.php

```diff
@@ -1,18 +1,10 @@
 <?php namespace PHRETS\Models;
 
-class RETSError
+class RETSError extends \Exception
 {
     protected $code;
     protected $message;
 
```

Comment:
*potential blocker:* Any reason to keep these around? Feels like a pit of failure.
```suggestion
```

Reply:
Yea, trying to minimize how much of the original library I modify.
As @mucha55 noted above
> we could eventually want to get our changes into PHRETS proper

Reply:
Which isn't to say removing those and further modifying this class would preclude that, but the goal of this PR was to make this implement `\Throwable`. I aimed to do that as minimally-invasively as possible.